contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/BasicFetchStrategy.java

```diff
@@ -64,21 +63,17 @@ public BasicFetchStrategy(HdfsFetcher fetcher, FileSystem fs, HdfsCopyStats stat
      * 'checkSumType' computed and returned. In case an error occurs during such
      * a copy, we do a retry for a maximum of NUM_RETRIES
      *
-     * @param fs
-     *            Filesystem used to copy the file
      * @param source
      *            Source path of the file to copy
      * @param dest
      *            Destination path of the file on the local machine
-     * @param stats
-     *            Stats for measuring the transfer progress
      * @param checkSumType
      *            Type of the Checksum to be computed for this file
      * @return A Checksum (generator) of type checkSumType which contains the
      *         computed checksum of the copied file
      * @throws IOException
      */
-    private CheckSum copyFileWithCheckSum(HdfsFile source, File dest, CheckSumType checkSumType) throws IOException {
+    private CheckSum copyFileWithCheckSum(HdfsFile source, File dest, CheckSumType checkSumType) throws Throwable {
```

Comment:
Ok, I verified the logs and I think it's safe to change the signatures and catches to ```IOException```, like you suggested.

I should merge this into master soon.

-F